index.html

```diff
@@ -6,11 +6,12 @@
     <title>The Frog Sayeth Croak</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
-    <script src="main.js"></script>
 </head>
 <body>
     <div class="container"></div>
-        <div id="home-container"></div>
-    
+        <div id="home"></div>
```

Comment:
We don't need a separate div here. You can add the id after the class on the first one.
`<div class="container" id="home">`

Reply:
YOU ARE RIGHT, GIRRL!